.github/workflows/tests.yml

```
@@ -0,0 +1,40 @@
+name: Run Tests
+
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.9', '3.10', '3.11', '3.12']
```

```suggestion
        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
```

We should include latest stable python version.